internal/indexcoord/index_coord.go

```diff
@@ -873,8 +873,10 @@ func (i *IndexCoord) watchMetaLoop() {
 	}
 }
 
-func (i *IndexCoord) tryAcquireSegmentReferLock(ctx context.Context, segIDs []UniqueID) error {
+func (i *IndexCoord) tryAcquireSegmentReferLock(ctx context.Context, buildID UniqueID, segIDs []UniqueID) error {
+	// IndexCoord use buildID instead of taskID.
 	status, err := i.dataCoordClient.AcquireSegmentLock(ctx, &datapb.AcquireSegmentLockRequest{
```

Comment:
question, do we ensure taskID of different?
IF not maybe nodeID + taskID can be the unique key